dist/shimmed-classes.txt

```diff
@@ -0,0 +1,272 @@
+com/nvidia/spark/rapids/AvoidAdaptiveTransitionToRow*
```

Comment:
Is there some kind of document that explains how this works?  Last we talked offline, I don't recall there being a need to explicitly list classes like this.

At a bare minimum there needs to be something in the contributing or dev docs that explains what people need to do if they are required to update these.  I'm hoping this is all temporary and we don't need to manually maintain these lists.

Reply:
the list was generated and probably tweaked manually a couple of times in the prototype  branch. 

We only use unshimmed-*.txt files in this PR. So we don't even need to include this file AFAICT, and when we come to dedupe part of the prototype we hopefully won't need it hardcoded but just generated. 

So in  this PR almost everything is shimmed and it should be easy to document the principle for unshimmed, i.e., because somehow publicly exposed either via documentation, Spark configuration before Plugin can do anything or just common dependencies.

Reply:
sorry just pushed some contributing docs

Reply:
this fiel actually isn't used right now.  I pulled it in from the original dev branch and just left thinking it might be used later, but I can remove from this.